### PR TITLE
Learn/module 7

### PR DESCRIPTION
**Pull request by akshaynr**
Implemented Formik and validations

Screenshot 2024-04-23 at 9 08 46 PM
Screenshot 2024-04-23 at 9 12 14 PM
Screenshot 2024-04-23 at 9 14 33 PM

### CODE REVIEW

**Comment by shubhamagarwal** [comment]
movies-app/src/components/MultiSelectField/MultiSelectField.css
do we need this file as this is empty ?

File: movies-app/src/components/MovieForm/MovieForm.js [hunk 1]
```diff
@@ -1,98 +1,127 @@
...
                             <label for="release_date" className="form-label">Release Date</label>
-                            <input type="text" className="form-control icon-input" id="release_date"   onChange={handleChange} name="release_date" value={formData.release_date} placeholder="Select Date" />
+                            <Field type="text" className="form-control icon-input" id="release_date" name="release_date" placeholder="Select Date" />
                             <div className="icon">
                                 <svg xmlns="http://www.w3.org/2000/svg" width="20" viewBox="0 0 448 512">
                                     <path fill="#e4534e" d="M112 0c8.8 0 16 7.2 16 16V64H320V16c0-8.8 7.2-16 16-16s16 7.2 16 16V64h32c35.3 0 64 28.7 64 64v32 32V448c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V192 160 128C0 92.7 28.7 64 64 64H96V16c0-8.8 7.2-16 16-16zM416 192H312v72H416V192zm0 104H312v80H416V296zm0 112H312v72h72c17.7 0 32-14.3 32-32V408zM280 376V296H168v80H280zM168 408v72H280V408H168zm-32-32V296H32v80H136zM32 408v40c0 17.7 14.3 32 32 32h72V408H32zm0-144H136V192H32v72zm136 0H280V192H168v72zM384 96H64c-17.7 0-32 14.3-32 32v32H416V128c0-17.7-14.3-32-32-32z" />
```
Comment (shubhamagarwal) [inline movies-app/src/components/MovieForm/MovieForm.js]: svg can be put in different file

Comment (akshaynr) [reply movies-app/src/components/MovieForm/MovieForm.js]: Svg moved to different file

**Comment by akshaynr** [comment]
> movies-app/src/components/MultiSelectField/MultiSelectField.css do we need this file as this is empty ?

Removed file

**Review by shubhamagarwal — APPROVED** [review]

**Pull request merged**